prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_CO_20240403.hl7

```diff
@@ -1 +1 @@
OBX|3|CWE|92142-9^Influenza virus A RNA [Presence] in Respiratory system specimen by NAA with probe detection^LN^^^^^^Flu A||260373001^Detected^SCT|||A^Abnormal^HL70078^^^^2.7|||W|||20240321220508+0000|12D4567890^Testing Lab^CLIA||Influenza SARS-CoV-2 (Flu SC2) Multiplex Assay_Centers for Disease Control and Prevention (CDC)^Influenza SARS-CoV-2 (Flu SC2) Multiplex Assay^^^^^^^Influenza SARS-CoV-2 (Flu SC2) Multiplex Assay_Centers for Disease Control and Prevention (CDC)|10190302000222^^DII|20240321222008+0000||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA
OBX|4|CWE|95419-8^Has symptoms related to condition of interest^LN^^^^2.69^^Has symptoms related to condition of interest||Y^Yes^HL70136|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
OBX|5|DT|11368-8^Illness or injury onset date and time^LN^^^^^^Illness or injury onset date and time||20240321000000+0000|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
OBX|6|CWE|82810-3^Pregnancy status^LN^^^^2.68^^Pregnancy status||60001007^Not pregnant^SCT|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
OBX|7|CWE|95418-0^Employed in a healthcare setting^LN^^^^2.69^^Employed in a healthcare setting||Y^Yes^HL70136|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
OBX|8|CWE|95421-4^Resides in a congregate care setting^LN^^^^2.69^^Resides in a congregate care setting||Y^Yes^HL70136|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
OBX|9|CWE|76691-5^Gender identity^LN^^^^^^Gender identity||446151000124109^Male gender identity^SCT|||^^^^^^2.7|||F|||20240321220508+0000|12D4567890^Testing Lab^CLIA||||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Houston^TX^77001^USA|||||QST
SPM|1|a375227a-751f-4035-8e0b-e81322f356c9&Testing Lab&12D4567890&CLIA^a375227a-751f-4035-8e0b-e81322f356c9&Testing Lab&12D4567890&CLIA||258467004^Nasopharyngeal washings^SCT^^^^2.67^^Nasopharyngeal washings||||87100004^Topography unknown (body structure)^SCT^^^^^^Topography unknown (body structure)|||||||||20240321220508+0000|20240321220508+0000

```

Comment:
The only difference in this file is at the end of `OBR|1|`. The value for this was `entered-in-error` because we were mapping _whatever_ was in DR.status to OBR.25 (a field which is a single character in the HL7 spec). This has been removed as OBRExtension isn't populated and so an unmapped value is simply lost in this scenario.